internal/controller/dpuoperatorconfig_controller_test.go

```diff
@@ -30,25 +53,14 @@ var (
 	testDpuDaemonName         = "dpu-daemon"
 	testSriovDevicePlugin     = "sriov-device-plugin"
 	testNetworkFunctionNAD    = "dpunfcni-conf"
-	testAPITimeout            = time.Second * 20
-	testRetryInterval         = time.Second * 1
+	testClusterName           = "dpu-operator-test-cluster"
+	testAPITimeout            = time.Second * 5
+	testRetryInterval         = time.Millisecond * 10
```

Comment:
On average, do you know how long it takes for an k8s API takes on kind? This seems a bit quick.

Reply:
It takes <1 sec.